packages/terra-icon/src/icon/IconConsultInstructionsForUse.jsx

```diff
@@ -1,14 +1,21 @@
 /* eslint-disable */
 import React from 'react';
 import IconBase from '../IconBase';
+import { FormattedMessage } from 'react-intl';
```

Comment:
react-intl will need to be added to the Icon package. I believe the current strategy is to add react-intl as a peerDependency

Reply:
updated here : https://github.com/cerner/terra-core/blob/dc294a9ec3f4f8825ecd45093a01510045db3e4a/packages/terra-icon/package.json#L32